### DISCUSSION TITLE
vue编写app.js的render函数无法渲染除了this.$slot.default的其他加入的

### DISCUSSION THREAD

**Discussion by lvsechoudaizi** [NervJS/taro#10141] [Q & A]
import Vue from 'vue'
import store from './store'
import TaroUiVue from 'taro-ui-vue'
import 'taro-ui-vue/dist/style/index.scss'
import UpdateNotice from './components/UpdateNotice.vue'
import './app.scss'
Vue.use(TaroUiVue)
// Vue.config.productionTip = false

const App = new Vue({
  store,
  onLaunch (options) {
    console.log(1)
  },
  components: {
    UpdateNotice,
  },
  render(h) {
    // this.$slots.default 是将要会渲染的页面
    // return h('block', this.$slots.default)
    var body   = this.$slots.default
    return h('view', [
      h(UpdateNotice, {}),  
      h('view', body)
    ])
  }
})

export default App

问题： 只渲染了声明的page页，UpdateNotice组件没有渲染出来
感谢回复！

**Comment by johnsoncheg** [comment]
@Chen-jj 这个问题有解么？

**Comment by Chen-jj** [comment]
@lvsechoudaizi @johnsoncheg 小程序不支持这样操作的，app.js 对应小程序的 app.js 文件，小程序的 app.js 是不能渲染东西的，渲染只能由页面进行。

**Reply by johnsoncheg** [reply]
了解，🙏
